### PR TITLE
Remove unnessary class

### PR DESCRIPTION
**Pull request by fearless-spider**
Fixes https://github.com/exercism/javascript/issues/1624

### CODE REVIEW

**Review by pertrai1 — APPROVED** [review]

File: exercises/practice/say/.meta/proof.ci.js [hunk 1]
```diff
@@ -1,90 +1,82 @@
...
-      15: 'fifteen',
-      16: 'sixteen',
-      17: 'seventeen',
-      18: 'eighteen',
-      19: 'nineteen',
-    };
+export const say = (n) => {
+  let result;
+  const number = { current: n };
+  let smallNumbers = {
```
Comment (junedev) [inline exercises/practice/say/.meta/proof.ci.js:4]: All the fixed objects and helper methods should be defined outside of the `say` function as `const`.

**Review by junedev — COMMENTED** [review]
Thanks for taking care of this! I checked the code and it still needs some tweaking to remove the nesting of the code, see comment.

**Comment by junedev** [comment]
@fearless-spider Thanks for the update, looks all good now.

**Review by junedev — APPROVED** [review]

**Comment by fearless-spider** [comment]
Awesome @junedev . I like this project, I will try to help as far as I can.

**Comment by junedev** [comment]
@fearless-spider Thanks! Unfortunately we don't have that many small and easy task open currently

**Comment by junedev** [comment]
@fearless-spider That's great to hear. Unfortunately a lot of the open tasks we have currently are rather big and difficult but keep an eye out for the task list.

**Pull request merged**